app/Http/Controllers/UserController.php

```diff
@@ -0,0 +1,155 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\User;
+use App\Role;
+use DB;
+use Session;
+use Hash;
+use Input;
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Namespaces.UseDeclaration.SpaceAfterLastUse</br>**Severity**: error</br>**File**: app/Http/Controllers/UserController.php</br>**Line**: 13</br>There must be one blank line after the last USE statement; 0 found;